### DISCUSSION TITLE
Not Detecting HDD in Bay 4 - Shorted Capacitors

### DISCUSSION THREAD

**Discussion by cbletz** [alexhorner/WD-DX4000-Installer#3] [Q&A] [answered]
Hey all,

On startup, my DX4000 spins up all 4 HDDs and the blue light comes on for each, but once I run "lsblk" in the command line I only see 3/4 of my 3TB volumes. All drives are working and have been wiped of all partitions. I took the unit apart and none of the pins on the SATA port looked bent. Could it be that the drive is getting power but there's something wrong with the data pins?

After some research it looks like I should check the BIOS to see if the SATA Controller is set to "AHCI" mode. It's my understanding that I'm unable to modify the BIOS without soldering in a Serial connection? Maybe it's a change in UEFI settings? I am very new to Linux so apologies if I'm missing something. 

Cheers

**Comment by alexhorner** [comment, marked as answer]
> Hey all,
> 
> On startup, my DX4000 spins up all 4 HDDs and the blue light comes on for each, but once I run "lsblk" in the command line I only see 3/4 of my 3TB volumes. All drives are working and have been wiped of all partitions. I took the unit apart and none of the pins on the SATA port looked bent. Could it be that the drive is getting power but there's something wrong with the data pins?
> 
> After some research it looks like I should check the BIOS to see if the SATA Controller is set to "AHCI" mode. It's my understanding that I'm unable to modify the BIOS without soldering in a Serial connection? Maybe it's a change in UEFI settings? I am very new to Linux so apologies if I'm missing something. 
> 
> Cheers

Hey there,

You do indeed need to solder to the serial port to change BIOS settings, yes.

There should be no need to do this however, because no settings should change.

If the 4th drive doesn't show up I would recommend you do the olde drive shuffle trick to see if any drive is capable of showing up in that slot.

If you're still having trouble, let me know and we can explore our options

**Comment by cbletz** [comment]
Thanks Alex,

Did some shuffling and none of the drives put in Bay 4 show up

**Reply by alexhorner** [reply]
The next thing I would try is using the original installation media to recover the original Windows environment. This is a fiddly process unless the hard drives are original to the unit. If they are you should be fine. Just no matter what you do, when you get to the part of the install process where it offers to install updates, say no to everything as it adds hours and hours to the install process

**Reply by cbletz** [reply]
Went through the "recreate storage" process and saw in the recovery log that it was only modifying 3/4 disks. Took the thing apart and found a group of 3 capacitors with a multi-meter that are shot. I'll survive with 3 bays but still a bummer. Thanks for the advice!

**Reply by alexhorner** [reply]
Nicely done! Make sure they aren't dragging the power supply down with a short, though I doubt you need me to tell you that.

Hopefully you can get it fixed. Please don't junk it, they're getting expensive to find!

**Reply by cbletz** [reply]
Yeah local repair shop quoted me for $300 and I got the unit for $100 - definitely keeping it though! Just might need to upgrade my drives sooner rather than later

**Reply by alexhorner** [reply]
Quoted you 300 for what exactly? Did they tell you what was broken?

**Reply by cbletz** [reply]
For replacing the 3 shorted capacitors. They are surface-mount so require a microscope, hot-air station, etc. otherwise I might attempt it myself. Going to try calling another place tomorrow just to be sure.

**Reply by alexhorner** [reply]
Good to know that's a thing that can happen. Curious. Definitely not a 300 job though. How big are they? Maybe you could attach a picture as a good reference?

**Reply by cbletz** [reply]
Apologies for the late reply! I finally got time to investigate further. The fault is actually in the IC just below the capacitors. I've attached a photo here with a button cell battery for scale:![photo_2022-08-05 15 15 04](https://user-images.githubusercontent.com/19161895/183006349-67dda372-8b4f-496d-be3c-059f4de25d29.jpeg)

[I found the schematic for the chip on the manufacturer's site.](https://www.richtek.com/Products/Switching%20Regulators/DC_DC%20StepDown%20Convertor/RT8250?sc_lang=en) Pin 4 is shorting to GND (pin 8).

Looks like a cheap replacement but I still don't have the experience or hot-air station to de-solder all 8 points at once.
